velox/functions/prestosql/aggregates/tests/MinMaxTest.cpp

```diff
@@ -364,17 +373,87 @@ TEST_F(MinMaxTest, row) {
 
   data = makeRowVector({
       makeRowVector({
+          makeFlatVector<StringView>({
+              "a"_sv,
+              "b"_sv,
+              "c"_sv,
+          }),
           makeNullableFlatVector<StringView>({
               "abc"_sv,
-              "ef"_sv,
               "efg"_sv,
+              "hij"_sv,
           }),
       }),
   });
-
   testAggregations({data}, {}, {"min(c0)", "max(c0)"}, {expected});
 }
 
+TEST_F(MinMaxTest, arrayCheckNulls) {
+  auto data = makeRowVector({
+      makeNullableArrayVector<int64_t>({
+          {1, 2},
+          {2, std::nullopt},
+          {6, 7},
+      }),
+  });
+
+  auto expectedMin = makeRowVector({
```

Comment:
Having these variables might be misleading because we do not expect a result, but an error.

Consider using PlanBuilder and AssertQueryBuilder directly.

```
auto plan = PlanBuilder().values({data}).singleAggregation({}, {"min(c0)"}).planNode();

  VELOX_ASSERT_THROW(AssertQueryBuilder(plan).copyResults(pool());
```

Reply:
Got it, thanks.

Reply:
Updated